util/dockernode.go

```diff
@@ -196,6 +203,62 @@ func (dn *DockerNode) setLatency(val string) error {
 	return cnet.SetLink(ifn, settings)
 }
 
+func (dn *DockerNode) setJitter(val string) error {
+	dur, err := time.ParseDuration(val)
+	if err != nil {
+		return err
+	}
+
+	ifn, err := dn.getInterfaceName()
+	if err != nil {
+		return err
+	}
+
+	settings := &cnet.LinkSettings{
+		Jitter: int(dur.Nanoseconds() / 1000000),
```

Comment:
FYI, Just now, I added a bit of documentation to the controlnet package: https://github.com/whyrusleeping/go-ctrlnet/blob/master/ctrlnet.go#L10